plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/OAuthMediatorPropertiesEditionPartForm.java

```diff
@@ -397,10 +399,11 @@ public void keyPressed(KeyEvent e) {
 
 	
 	protected Composite createUsernameText(FormToolkit widgetFactory, Composite parent) {
-		createDescription(parent, EsbViewsRepository.OAuthMediator.Properties.username, EsbMessages.OAuthMediatorPropertiesEditionPart_UsernameLabel);
-		username = widgetFactory.createText(parent, ""); //$NON-NLS-1$
+	    filterCredentialsSubPropertiesGroup = EEFPropertyViewUtil.createSubsectionGroup(widgetFactory, parent, "Credentials", true);
```

Comment:
Correct formatting.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/806